contracts/common/ContractRecovery.sol

```diff
@@ -68,7 +70,7 @@ contract ContractRecovery is ContractRecoveryDataTypes, CommonStorage { // ignor
     recoveryApprovalCount = 0;
     recoveryEditedTimestamp = block.timestamp;
 
-    emit RecoveryModeEntered();
+    emit RecoveryModeEntered(msg.sender);
```

Comment:
Isn't this information already available on the tx?

Reply:
I originally didn't have this, but ended up including it - `tx.origin` is available, but if the permission holder is a contract, it is not easily available.